src/witan/models/dem/ccm/components_functions.clj

```diff
@@ -2,16 +2,23 @@
   (:require [incanter.core :as i]
             [clojure.core.matrix.dataset :as ds]
             [witan.datasets :as wds]
-            [schema.core :as s]))
+            [schema.core :as s]
+            [incanter.stats :as st]
+            [clojure.string :as str]))
 
 ;; Calculate rates/values for alternative ways to project components of change:
 
 ;; Calculate averages for fixed values/rates or trends:
 (defn jumpoffyr-method-average
   "Takes in a dataset with historical data, a column name to be averaged,
-  a name to rename the average column, a number of years to average on
-  and the jumpoff year. Returns a dataset where the column to be averaged contains
+  a name to rename the average column, and years to start and end averaging for.
+  Returns a dataset where the column to be averaged contains
+
   the averages for all the years of interest and is renamed to avg-name."
+
+  ;;@eleonore is this definitely what happens? ^^^  I thought an additional
+  ;;col was added as below?
+
```

Comment:
@seb231: The `rollup` function used here either add a column or replace a column when the column name passed in (as the 3rd parameter) already exists in the dataset. 
Here we pass in a column name that is already in the dataset so its values get replaced.